src/components/Search/SearchInput.jsx

```diff
@@ -1,58 +1,47 @@
 import React from 'react';
 
-import Container from '@mui/material/Container';
-import FormControl from '@mui/material/FormControl';
-import InputLabel from '@mui/material/InputLabel';
-import MenuItem from '@mui/material/MenuItem';
-import Select from '@mui/material/Select';
+import Grid from '@mui/material/Grid';
 import TextField from '@mui/material/TextField';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 import SearchViewHelper from '../../utils/searchViewHelper';
 import SearchResultsList from './SearchResultsList';
 
-const { handleChangeDropdown, handleChangeSearch } = SearchViewHelper;
+const { handleChangeSearch } = SearchViewHelper;
 
 function SearchInput(props) {
-  const { repository, searchResults } = props;
-  const repoList = ['try', 'autoland', 'mozilla-central'];
+  const { inputError, inputHelperText, searchResults } = props;
 
   return (
-    <Container maxWidth="md">
-      <FormControl sx={{ width: '25%' }}>
-        <InputLabel id="select-repository">repository</InputLabel>
-        <Select
-          value={repository}
-          labelId="select-repository"
-          label="repository"
-        >
-          {repoList.length > 0 &&
-            repoList.map((item) => (
-              <MenuItem
-                id={item}
-                value={item}
-                key={item}
-                onClick={handleChangeDropdown}
-              >
-                {item}
-              </MenuItem>
-            ))}
-        </Select>
-      </FormControl>
-      <TextField
-        label="Search By Revision ID or Author Email"
-        variant="outlined"
-        sx={{ width: '75%' }}
-        onChange={handleChangeSearch}
-      />
+    <Grid item xs={6}>
+      {!inputError && !inputHelperText ? (
+        /* text field without errors */
+        <TextField
+          label="Search By Revision ID or Author Email"
+          variant="outlined"
+          sx={{ width: '100%' }}
+          onChange={handleChangeSearch}
+        />
+      ) : (
+        /* text field with errors */
+        <TextField
+          error
+          helperText={inputHelperText}
+          label="Search By Revision ID or Author Email"
+          variant="outlined"
+          sx={{ width: '100%' }}
+          onChange={handleChangeSearch}
+        />
+      )}
```

Comment:
The same logic could be applied here, although it's less important here because the JSX is much smaller. So I'll leave up to you here.